docs/DriverParseableOutput.rst

```diff
@@ -76,13 +82,18 @@ key. An "outputs" object will have two fields, a "kind" describing the type of
 the output, and a "path" containing the path to the output. A "began" message
 will specify the command which was executed under the "command_executable" and
 "command_arguments" keys.
+To get the real process identifier for the process that started, get the value
+of "process.real_pid".
 
 Example::
 
    {
      "kind": "began",
      "name": "compile",
      "pid": 12345,
+     "process": {
+       "real_pid": 12345
```

Comment:
`pid` and `real_pid` have some potential to create confusion, but I don't see answer, and it's not your fault, anyway.

Reply:
The main problem here is that 'pid' is re-used, but I also don't see another solution without breaking infrastructure that relies on this.

Reply:
Yes, sigh.